testsuite/gnat2goto/testsuite.py

```diff
@@ -74,19 +74,26 @@ def collect_result_including_uok(name, process, _job_info):
             test_status = test_result[0].split(':')[0]
             if test_status == 'UOK':
                 test_metrics['uok'] += 1
+            elif test_status == 'INVALID_TEST':
+                test_metrics['invalid'] += 1
 
     run_testcase = generate_run_testcase('run-test', discs, m.options)
 
     MainLoop(test_list, run_testcase, collect_result_including_uok, m.options.mainloop_jobs)
 
     print "Summary: Ran %(run)s/%(total)s tests, with %(failed)s failed, %(crashed)s crashed, %(uok)s unexpectedly passed." % test_metrics
```

Comment:
Could you include the 'invalid' count in the Summary line printed here.

Reply:
Hm, I deliberately excluded it because an invalid test is more of an exception rather than a test result, in my mind they were two different things.

Reply:
I personally didn't see that line so much as "test results" and more "one line summary of what happened". Not a big deal either way, but its sometimes nice to have that summary.